README.md

```diff
@@ -1,2 +1,44 @@
-# portfolio
-My portfolio website
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# My Portfolio
+
+> Developing Portfolio page from scratch.
```

Comment:
- [ OPTIONAL ] I think it would be great if you add here more description to your project to explain what is about 🚀💯